src/extension/netcdf-out/src/test/java/org/geoserver/wcs2_0/WCSNetCDFMosaicTest.java

```diff
@@ -205,9 +204,9 @@ private void setupNetCDFoutSettings(QName name) {
     private void setupNetCDFoutSettings(QName name, boolean setNoData) {
         CoverageInfo info = getCatalog().getCoverageByName(getLayerId(name));
 
-        // Set the Declared SRS
-        info.setSRS("EPSG:4326");
-        info.setProjectionPolicy(ProjectionPolicy.REPROJECT_TO_DECLARED);
+        //        // Set the Declared SRS
```

Comment:
Are these comments intentionally left here?

Reply:
Nope, I should have removed the lines, doing so.